media/js/firefox/sync.js

```diff
@@ -126,6 +126,12 @@
         'data-download-version': 'Firefox for Android'
     });
 
+    // Firefox Sync sign in flow button
+    $('#cta-sync').on('click', function(e) {
+        e.preventDefault();
+        Mozilla.UITour.showFirefoxAccounts();
```

Comment:
hmmm, I wonder perhaps we can test that it goes to about:accounts?action=signup&entrypoint=uitour - will try it out.